CHANGELOG.md

```diff
@@ -5,6 +5,48 @@ All notable changes to Zebra are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.3.0](https://github.com/ZcashFoundation/zebra/releases/tag/v1.3.0) - 2023-09-25
+
+This release provides support for the Spend before Sync synchronization
```

Comment:
Since this feature has gone by 3 different names, some users might not know what it's for:

```suggestion
This release provides support for the Spend before Sync light wallet synchronization
```